packages/frontend/src/config/flags.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * List of all supported Launch Darkly flags on frontend.
+ */
+/**
+ * Display flags
+ */
+export const BANNER_DISPLAY_FLAG = 'banner_display'
```

Comment:
nit: technically not a flag. Maybe we call it `BANNER_MESSAGE` or `BANNER_DISPLAY_MESSAGE` or something?